packages/account-abstraction-kit/src/AccountAbstraction.ts

```diff
@@ -76,12 +76,6 @@ class AccountAbstraction {
     })
 
     try {
-      await getSafeContract({
```

Comment:
This line is necessary, it is a workaround to detect if the Safe is deployed

Reply:
Okay, I see. I'll close the PR